src/FinalFrame.js

```diff
@@ -0,0 +1,22 @@
+class FinalFrame extends Frame {
+  constructor() {
+    super();
```

Comment:
Not sure what this is for?

Reply:
@Battery0
 Goes with the above ^ When you inherit from a class, you can call super in the constructor to first execute the constructor of the parent. Most languages have a way of doing this - take a look at it!